packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/sticky-toc.twig

```diff
@@ -1,5 +1,5 @@
 {% if stickyTOC %}
-<section class="ma__sticky-toc" data-min-to-show="{{ stickyTOC.min ? stickyTOC.min : 2 }}" data-parent="{{ stickyTOC.parent }}"
+<section class="ma__sticky-toc" data-min-to-show="{{ stickyTOC.min }}" data-parent="{{ stickyTOC.parent }}"
```

Comment:
As removing `? stickyTOC.min : 2`, there is no default value for `data-min-to-show`.  It seems this is causing to `data-min-to-show` to be empty in Drupal.